diff --git a/node_modules/node-html-to-image/src/index.js b/node_modules/node-html-to-image/src/index.js
index f7d3d4f..ad877a3 100644
--- a/node_modules/node-html-to-image/src/index.js
+++ b/node_modules/node-html-to-image/src/index.js
@@ -19,7 +19,7 @@ module.exports = async function(options) {
   const cluster = await Cluster.launch({
     concurrency: Cluster.CONCURRENCY_CONTEXT,
     maxConcurrency: 2,
-    puppeteerOptions: { ...puppeteerArgs, headless: true },
+    puppeteerOptions: { puppeteerArgs: {args: ['--no-sandbox', '--disable-gpu']}, headless: true }
   });
 
   let buffers = []
